spec/controllers/callbacks_controller_spec.rb

```diff
@@ -37,5 +37,61 @@
         expect(session).not_to be_empty
       end
     end
+
+    context 'with existing user with invalid token' do
+      let(:user) { create :user_with_invalid_token }
+      before do
+        request.env['omniauth.auth'] = mock_auth(user)
+        request.env['devise.mapping'] = Devise.mappings[:user]
+      end
+
+      it "doesn't create a new User object" do
+        expect do
+          post :github
+        end.to change { User.count }.by(0).and raise_error(MissingMembershipError) { |error|
```

Comment:
This feels weird — why would you be OK with invalid token to return `MissingMembershipError` of all errors, and even specify error message to be `'You are not in an organisation!'`? When you clearly know, that this is not what happened?!